apps/server/src/modules/h5p-editor/service/h5p-player.service.ts

```diff
@@ -3,18 +3,19 @@ import { H5PPlayer } from '@lumieducation/h5p-server';
 import { ContentStorage } from '../contentStorage/contentStorage';
 import { LibraryStorage } from '../libraryStorage/libraryStorage';
 import { h5pConfig, h5pUrlGenerator } from './config/h5p-service-config';
+import { Translator } from './h5p-translator.service';
 
 export const H5PPlayerService = {
 	provide: H5PPlayer,
 	inject: [ContentStorage, LibraryStorage],
-	useFactory: (contentStorage: ContentStorage, libraryStorage: LibraryStorage) => {
+	useFactory: async (contentStorage: ContentStorage, libraryStorage: LibraryStorage) => {
 		const h5pPlayer = new H5PPlayer(
 			libraryStorage,
 			contentStorage,
 			h5pConfig,
 			undefined,
 			h5pUrlGenerator,
-			undefined,
+			await Translator.translate(),
```

Comment:
same as above

Reply:
@MartinSchuhmacher Also in this solution, an 'await' is needed inside a const declaration:
`const translator = await Translator.translate();`